spec/controller/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] The same here. Your post spec file should include tests that ensure the correct view template is rendered. Example: 
```ruby
    it "renders 'index' template" do
       expect(response).to render_template(:index)
    end
```
- Kindly add this ✍🏻